controllers/suite_test.go

```diff
@@ -94,6 +94,9 @@ var _ = BeforeSuite(func(done Done) {
 	err = solrv1beta1.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
+	err = solrv1beta1.AddToScheme(scheme.Scheme)
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
Isn't this a duplicate of L94-95 ?

Reply:
Yeah, good find, bunch of duplicates, ill clean up!